test/io/ferguson/buffer_test.chpl

```diff
@@ -1,35 +1,60 @@
 use Buffers;
 
+proc testassign() {
+  var b = new bytes(50);
+  {
+    // test assignment overload.
+    var b_tmp:bytes;
+    var b_tmp2 = new bytes();
+    b_tmp2 = b;
+    b_tmp = b;
+  }
+}
+
 proc main() {
+  testassign();
+
   // allocate 50 bytes
   var b = new bytes(50);
-
+  {
+    // test assignment overload.
+    var b_tmp:bytes;
+    var b_tmp2 = new bytes();
+    b_tmp2 = b;
+    b_tmp = b;
+  }
```

Comment:
In some of the (wrong) configurations of Buffers.chpl, I was seeing the error here but not in testassign. So I left them both in.